index.js

```diff
@@ -0,0 +1,36 @@
+const express = require("express");
+const cors = require("cors");
+const bcrypt = require("bcryptjs");
+
+const db = require("./data/dbConfig.js");
+const PORT = 4200;
+
+const server = express();
+
+server.use(express.json(), cors());
+
+// | POST   | /api/register | Creates a `user` using the information sent inside the `body` of the request. Hashes the password
+
+server.post("./api/register", (req, res) => {
+  const user = req.body;
```

Comment:
I would recommend destructuring this object into an appropriate variable but this is just preference.